sql/server_component/clone_protocol_service.cc

```diff
@@ -539,6 +545,13 @@ DEFINE_METHOD(int, mysql_clone_get_response,
   *net_length = 0;
   *length = my_net_read(net);
 
+  /** The ZSTD object has been updated to include the newly created decompressor
```

Comment:
This is not a Doxygen comment:
```suggestion
  /* The ZSTD object has been updated to include the newly created decompressor
```